dev/.scalafmt.conf

```diff
@@ -22,6 +22,13 @@ align.tokens = []
 optIn = {
   configStyleArguments = false
 }
-danglingParentheses = false
-docstrings = JavaDoc
+danglingParentheses.preset = false
+docstrings.style = Asterisk
 maxColumn = 98
+runner.dialect = scala212
+fileOverride {
+  "glob:**/src/**/scala-2.13/**.scala" {
+    runner.dialect = scala213
+  }
+}
```

Comment:
Error formatting Scala files: Invalid config: Default dialect is deprecated; use explicit: [sbt0137,sbt1,scala211,scala212,scala212source3,scala213,scala213source3,scala3]
[ERROR] Also see https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects"